### PR TITLE
[FrameworkBundle] Remove default value for `gc_probability` config option

### LINKED ISSUE #20196: [FrameworkBundle] Remove default value for `gc_probability` config opti…
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#58165
| PR author(s) | @nicolas-grekas
| Merged in    | 7.2

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 7.2 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by raphael-geffroy**
Fixes #20196

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -1852,6 +1852,8 @@ started on every session initialization. The probability is calculated by
 using ``gc_probability`` / ``gc_divisor``, e.g. 1/100 means there is a 1%
 chance that the GC process will start on each request.
 
+If not set, ``php.ini``'s `session.gc_probability`_ directive will be relied on.
```
Comment (MrYamous) [inline reference/configuration/framework.rst:1855]: Maybe with a versionadded directive to indicate this is a new behavior ? 
And we also need to remove default value few lines above

Comment (raphael-geffroy) [reply reference/configuration/framework.rst:1855]: Thanks for the feedback ! Just committed your recommandations.

**Comment by javiereguiluz** [comment]
Merged! Raphaël thanks a lot for this contribution ... and congrats on your first Symfony Docs contribution 🎉

**Comment by raphael-geffroy** [comment]
> Merged! Raphaël thanks a lot for this contribution ... and congrats on your first Symfony Docs contribution 🎉 

Thank you for pointing me those 'good first issues' ! I'm glad I could take a first step in the symfony-docs contribution 😉

**Pull request merged**